src/Algebra/Properties/Semiring/Divisibility.agda

```diff
@@ -0,0 +1,32 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some basic items and theory for Semiring.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra using (Semiring)
+import Algebra.Properties.Monoid.Divisibility as MonoidDiv
+open import Data.Product using (_,_)
+
+module Algebra.Properties.Semiring.Divisibility {r₁ r₂} (R : Semiring r₁ r₂) where
+
+open Semiring R
+open import Algebra.Divisibility _≈_ _*_ using (∣-max)
+open MonoidDiv *-monoid public
```

Comment:
I set the comments like
```
-- Re-exporting contents of divisibility over monoids
--
open MonoidDiv *-monoid public renaming (ε∣_ to 1∣_)
```
This looks better because the dashes line inside a sequence the `open` declarations makes it difficult to read.

If you write that you insist on your version, then I shall change it to your version (or you do this yourself).

Reply:
Please stick to the style that is used everywhere else throughout the library and do not invent your own!

Reply:
Sorry, I have forgotten to return these headings to your style.
Let us do this in the next commit.